pvlib/soiling.py

```diff
@@ -69,15 +69,15 @@ def hsu(rainfall, cleaning_threshold, tilt, pm2_5, pm10,
 
     # never use mutable input arguments
     if depo_veloc is None:
-        depo_veloc = {'2_5': 0.004, '10': 0.0009}
+        depo_veloc = {'2_5': 0.0009, '10': 0.004}
 
     # accumulate rainfall into periods for comparison with threshold
     accum_rain = rainfall.rolling(rain_accum_period, closed='right').sum()
     # cleaning is True for intervals with rainfall greater than threshold
     cleaning_times = accum_rain.index[accum_rain >= cleaning_threshold]
 
     horiz_mass_rate = pm2_5 * depo_veloc['2_5']\
-        + np.maximum(pm10 - pm2_5, 0.) * depo_veloc['10']
+        + np.maximum(pm10 - pm2_5, 0.) * depo_veloc['10'] * 3600
```

Comment:
Sorry if I'm thinking about this wrong, but I want to raise two points about this calculation:
- Is it correct to hardcode the 3600?  Shouldn't it be based on the datetime index's interval?  I'm not seeing any documentation requiring the input timeseries to be hourly. 
- Regardless of whatever value that timescale coefficient takes, shouldn't it apply to both particulate sizes?  Right now it only affects pm10.

Reply:
@kanderson-nrel: I agree that I should take another look at the hardcoding of 3600.  About your second point, let me look more carefully at the Matlab version and see how it deals with it.